docs/before/account_olcf.md

```diff
@@ -0,0 +1,151 @@
+# OLCF Access Guide
+
```

Comment:
Should we add a "Quicklinks" cell here that lists the useful links explained below? Something similar to what is done in the LCLS page, like:
```
!!! note "Quicklinks"
    - **More info** - [OLCF User guide][2]
    - ...
```